src/current/v25.4/configure-logs.md

```diff
@@ -77,10 +79,47 @@ capture-stray-errors: ... # parameters for the stray error capture system
 
 Providing a logging configuration is optional. Any fields included in the YAML payload will override the same fields in the [default logging configuration](#default-logging-configuration).
 
-{{site.data.alerts.callout_success}}
-You can view the default settings by running `cockroach debug check-log-config`, which returns the YAML definitions and a URL to a visualization of the default logging configuration.
+{{site.data.alerts.callout_danger}}
+When you provide a minimal logging configuration for `file-groups`, CockroachDB replaces the entire default `file-groups` structure rather than merging only the specified fields. As a result, all logging channels flow into the `default` file sink, which can lead to noisy logs and rapid log rotation. Although users may expect partial YAML overrides to merge with the default logging configuration, this behavior is not supported. To avoid misconfiguration, validate logging settings using the [`cockroach debug check-log-config`](#cockroach-debug-check-log-config) command.
```

Comment:
Can we just say partial YAML overrides that will merge with the default logging configuration is not supported?